docs(typo): modify Compile-time Type Checking Section

GwonHeeJun opened the pull request:
- Before : `// Argument of type '"#00ffzaz"' is not assignable to parameter
of type '...'.ts(2345)`
- After : `// Argument of type '"#00ffzz"' is not assignable to parameter
of type '...'.ts(2345)`

Change the wrong typo.

junhoyeo commented:
Hi, @GwonHeeJun! Thanks for contributing. ☺️
I had really no idea of that typo and wouldn't have found it without your help.
These are my feedback;

1. Can you change the comment, not the code? The intended example(`#00ffzz`) is to show that it detects wrong digits like `z`, even it has the same length as `#00fffa`. I'll be hoping you to change the content in your commit as well.
2. This is an somewhat minor request, and it's about this our project's commit convention. Maybe it'll be nicer if you remove the spaces before the parentheses in your commit message/PR title?

GwonHeeJun commented:
Can I change it like this? @junhoyeo 

```
- useColor('#00ffzz')
// Argument of type '"#00ffzz"' is not assignable to parameter of type '...'.ts(2345)
```

junhoyeo commented:
> Can I change it like this? @junhoyeo

Yup. That will be wonderful 🎉

GwonHeeJun committed:
docs(typo): modify Compile-time Type Checking Section

Before : // Argument of type '"#00ffzaz"' is not assignable to parameter
of type '...'.ts(2345)
After : // Argument of type '"#00ffzz"' is not assignable to parameter
of type '...'.ts(2345)

Change the wrong typo.

GwonHeeJun commented:
> > Can I change it like this? @junhoyeo
> 
> Yup. That will be wonderful 🎉

I fixed it! Check it out plz 😎 @junhoyeo

junhoyeo reviewed (approved):
Nice 👍👍👍
